adapters/koa/koa.ts

```diff
@@ -1,16 +1,19 @@
 import * as Router from 'koa-router';
-import * as Koa from 'koa';
+import { ParameterizedContext } from 'koa';
 import * as runtime from '@smartlyio/oats-runtime';
 
-function adapter(router: Router): runtime.server.ServerAdapter {
+function adapter<StateT, CustomT, RequestContext extends object>(
```

Comment:
does requestcontext really need to extend object?

Reply:
Not really, can remove that.